tests/test_core.py

```diff
@@ -35,6 +36,11 @@ def no_wasm(f):
 def no_wasm_backend(f):
   return skip_if(f, 'is_wasm_backend')
 
+def no_wasm_backend_note(note=''):
```

Comment:
Got it. I just had to look up the behavior of decorators with vs. without arguments; it's really weird.